Sources/MongoSwift/MongoClient.swift

```diff
@@ -274,4 +276,10 @@ public class MongoClient {
     public func db(_ name: String, options: DatabaseOptions? = nil) -> MongoDatabase {
         return MongoDatabase(name: name, client: self, options: options)
     }
+
+    /// Executes an `Operation` using this `MongoClient` and an optionally provided session.
+    internal func executeOperation<T: Operation>(_ operation: T,
+                                                 session: ClientSession? = nil) throws -> T.OperationResult {
+        return try self.executor.execute(operation, client: self, session: session)
```

Comment:
not sure where we landed on this, but is the label for `session` superfluous here?

Reply:
I personally try to limit how long I think about internal parameter stuff, so I don't really feel strongly either way also (I spent _way_ too long thinking about what to call the error getters). My general approach these days is to choose something acceptable that I can update either in code review if someone else does feel strongly one way or, if after using it in the code for a while another way feels more natural, just refactor it.